File: 백준/Bronze/1357. 뒤집힌 덧셈/뒤집힌 덧셈.cc
#include <iostream>
#include <string>
using namespace std;

int main() {
    ios::sync_with_stdio(false); cin.tie(NULL); cout.tie(NULL);
    
    string x, y, sum;
    int int_sum;
    string reversed_x = "", reversed_y = "", reversed_sum = "";
    cin >> x >> y;

    for (int i = 0; i < x.length(); i++)
        reversed_x = x[i] + reversed_x;

    for (int i = 0; i < y.length(); i++)
        reversed_y = y[i] + reversed_y;

    int_sum = stoi(reversed_x) + stoi(reversed_y);
    sum = to_string(int_sum);

    for (int i = 0; i < sum.length(); i++)
        reversed_sum = sum[i] + reversed_sum;

    cout << stoi(reversed_sum) << endl;
    
    
    return 0;
}

File: 백준/Bronze/1357. 뒤집힌 덧셈/뒤집힌 덧셈.py
import sys

x, y = map(str, sys.stdin.readline().split())
reversed_x = ""
reversed_y = ""
reversed_sum = ""

for i in x:
    reversed_x = i + reversed_x

for i in y:
    reversed_y = i + reversed_y

sum = str(int(reversed_x) + int(reversed_y))

for i in sum:
    reversed_sum = i + reversed_sum

sys.stdout.write(str(int(reversed_sum)))